tests/fsyacc/Test1/test1.fsproj

```diff
@@ -3,18 +3,16 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <OtherFlags>--mlcompatibility</OtherFlags>
+    <NoWarn>FS0760</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="../lexing.fs" />
-    <Compile Include="../parsing.fs" />
     <Compile Include="../../../src/Common/Arg.fs" />
-    <Compile Include="../arg.fs" />
-    <Compile Include="../tree.ml" />
+    <Compile Include="../arg2.fs" />
```

Comment:
Funny finding - two files with the same name did compile fine but displayed incorrectly(only one of them was displayed in solution explorer) in VS2022. I decided to rename the one form tests folder to `arg2` to make it easier to play with the tests projects in VS2022.